scripts/west_commands/runners/stm32cubeprogrammer.py

```diff
@@ -169,7 +169,7 @@ def do_add_parser(cls, parser):
             # is displayed when an invalid value is provided for this argument.
             type=functools.wraps(int)(lambda s: int(s, base=0)),
             required=False,
-            help="Address where flashing should be done"
+            help="Flashing location address. To be used only with .bin files"
```

Comment:
```suggestion
            help="Flashing location address. Required to flash .bin files, ignored otherwise."
```

I think the following wording could be better:
```suggestion
            help="Address where binary should be flashed. Required to flash .bin files, ignored otherwise."
``````